package.json

```diff
@@ -7,12 +7,10 @@
   "repository": "git@github.com:comit-network/comit-js-sdk.git",
   "author": "CoBloX developers <team@coblox.tech>",
   "scripts": {
-    "check": "tsc --noEmit",
+    "check": "tsc --noEmit && tslint -p . && prettier --check '**/*.{ts,js,json,yml}'",
     "rebuild": "rimraf dist/ && tsc",
-    "gen": "mkdir -p ./gen && json2ts -i ./siren.schema.json -o ./gen/siren.d.ts",
+    "postinstall": "mkdir -p ./gen && json2ts -i ./siren.schema.json -o ./gen/siren.d.ts && prettier --write ./gen/siren.d.ts",
```

Comment:
:question: Why did you call it `postinstall`?

Reply:
so that it's run automatically after `install`.